packages/overlay/src/index.tsx

```diff
@@ -39,28 +39,35 @@ function createStyleSheet(styles: string) {
 /**
  * Open the Spotlight debugger Window
  */
-export async function openSpotlight(path?: string | undefined) {
```

Comment:
h: this is publicly exported, right? This is semver-breaking if we change from Promise<void> to void

Reply:
Ehh, can revert. Just, there was no reason for these to be async.

Reply:
yeah good point! I mean we can also just cut a major of the overlay package 😅 but it probably makes sense to wait for a change that justifies a major, I'll let you be the judge of that 😅